### ISSUE TITLE
onPageChange

### ISSUE THREAD

**Issue by simply-news** [darenju/react-native-flip-page#15]
Hello, 

Is there an onPageChange attribute we can use?

Thank you! Big fan of your react web version one btw!

**Comment by darenju** [comment]
No, as of now, there is no `onPageChange` callback on the native version…

You can try to contribute as I don’t have much time to work on OSS for now.
